src/app/containers/FeatureIndexMain/index.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jsx-a11y/aria-role */
+import React from 'react';
+import path from 'ramda/src/path';
+import { frontPageDataPropTypes } from '#models/propTypes/frontPage';
```

Comment:
If both, Front pages and Feature Indexes, are going to use the same propTypes, should we rename it to a more generic name?

Reply:
I think so! I didn't want to make too many changes before running through this PR with people :)

Reply:
Although maybe it's best to have a separate featureIndexDataPropTypes that is made up of the same constituent parts to avoid coupling them together? It feels a little odd to keep the components separate to allow for the two to diverge but then not doing the same for the propTypes

Reply:
That would make more sense 👍